src/corner_kick/src/components/Portal/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file defines a React component which adds other elements
+ * in the main area of the application
+ */
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { UnknownPortalException } from 'SRC/utils/exceptions';
+
+/**
+ * The area where we wish to portal content to
+ *
+ * These areas include:
+ * -   **main:** Contains the primary focus of the page
+ * -   **sidebarTitle:** Contains the page title and additional options
+ *     specific to the page
+ * -   **sidebar:** Contains additional controls or information relevant
+ *     to the current page
+ * -   **sidebarControl:** Contains navigation controls to switch between pages
+ * -   **console:** Contains views visible across the application
+ * -   **footer(Left|Right):** Contains simple breadcrumbs of information
```

Comment:
Just curious, is "breadcrumb" a technical web term? Or is it just being used in the normal sense here?

Reply:
Used in the formal sense here. A small series of connected information relating to the current view. Think "are we connected to ROS?" "How many shapes are there?" etc.